hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/SignerManager.java

```diff
@@ -17,14 +17,19 @@
  */
 package org.apache.hadoop.fs.s3a;
 
-import com.amazonaws.auth.Signer;
-import com.amazonaws.auth.SignerFactory;
 import java.io.Closeable;
 import java.io.IOException;
+import java.util.LinkedList;
```

Comment:
This class has only gone in. I thought it was going into .impl, but clearly not

1. Move in this patch to a subdir; s3a.auth is the most appropriate
1. Sort out the imports

I don't want any new implementation classes to go into fs.s3a; I have a goal of marking the impl as private in java 11 modules. Please avoid adding things there unless unavoidable -and explain to have to justify. thx

Reply:
Moved to s3a.auth.

What needs to change in the imports?
It's using
```
java.*
\n
Everything other than org.apache.*
\n
org.apache.*
\n
static imports
```